src/renderer/families/solana/shared/fields/ValidatorsField.js

```diff
@@ -28,13 +25,9 @@ type Props = {
 };
 
 const ValidatorField = ({ t, account, onChangeValidator, chosenVoteAccAddr, status }: Props) => {
-  if (!status) return null;
-
   invariant(account && account.solanaResources, "solana account and resources required");
 
-  const { solanaResources } = account;
-
-  const [search, setSearch] = useState("");
+  const [search] = useState("");
```

Comment:
Removed the setter for search, which is weird. Is it not weird that it's not used ?

Reply:
Probably either some forgotten or abandoned design. No one complained so :+1: to remove.